common/containers/Tabs/Swap/components/SwapProgress.jsx

```diff
@@ -40,22 +40,15 @@ export default class SwapProgress extends Component {
       if (!hasShownViewTx) {
         let linkElement;
         let link;
+        const notificationMessage = translate('SUCCESS_3', true) + outputTx;
         // everything but BTC is a token
         if (destinationKind !== 'BTC') {
           link = bityConfig.ethExplorer.replace('[[txHash]]', outputTx);
-          linkElement = (
-            <a href={link} target="_blank" rel="noopener">
-              {' '}View your transaction{' '}
-            </a>
-          );
+          linkElement = `<a href="${link}" target='_blank' rel='noopener'>${notificationMessage}</a>`;
           // BTC uses a different explorer
         } else {
           link = bityConfig.btcExplorer.replace('[[txHash]]', outputTx);
-          linkElement = (
-            <a href={link} target="_blank" rel="noopener">
-              {' '}View your transaction{' '}
-            </a>
-          );
+          linkElement = `<a href="${link}" target='_blank' rel='noopener'>${notificationMessage}</a>`;
```

Comment:
Is this going to be alright? Did we switch back to setInnerHtmlDangerously?

Reply:
No, there's no setInnerHtmlDangerously used here.

Reply:
The user will see `<a href...` on their screen when it's rendered here: https://github.com/MyEtherWallet/MyEtherWallet/blob/c7146a3d728684a9b77bc3f6b161dcb27dad9b75/common/containers/App/Notifications.jsx#L38

https://facebook.github.io/react/docs/dom-elements.html#dangerouslysetinnerhtml for more information